packages/@sanity/imagetool/src/HotspotImage.js

```diff
@@ -136,7 +138,7 @@ export default class HotspotImage extends React.PureComponent {
       }
     })
     return (
-      <div className={className} style={style} ref={this.setContainerElement}>
+      <div className={className || styles.root} style={style} ref={this.setContainerElement}>
```

Comment:
Shouldn't we merge the classNames? That's the pattern we use elsewhere I think.

Reply:
👍